pkg/api/plugins.go

```diff
@@ -33,6 +37,14 @@ import (
 	"github.com/grafana/grafana/pkg/web"
 )
 
+// pluginsCDNFallbackRedirectRequests is a metric counter keeping track of how many
+// requests are received on the plugins CDN backend redirect fallback handler.
+var pluginsCDNFallbackRedirectRequests = promauto.NewCounterVec(prometheus.CounterOpts{
```

Comment:
Since we have a service for plugins cdn, I'm wondering it we should move this into `pluginscdn.Service` directly?

Reply:
Yeah same as other - I think is okay here for now!